apps/nextjs-website/src/app/profile/personal-data/page.tsx

```diff
@@ -109,27 +109,30 @@ const PersonalData = () => {
     const isEditing = editItem?.name === item.name;
 
     return (
-      <Box key={index}>
-        {isPasswordItem && (
-          <PasswordFormWrapper
-            item={item}
-            isEditing={isEditing}
-            onCancel={() => setEditItem(null)}
-            onSave={handleChangePassword}
-            onEdit={() => setEditItem(item)}
-          />
-        )}
-        {isEmailItem && (
-          <EmailFormWrapper
-            item={item}
-            isEditing={isEditing}
-            onCancel={() => setEditItem(null)}
-            onSave={handleChangeEmail}
-            onEdit={() => setEditItem(item)}
-          />
-        )}
-        {showDivider && <Divider />}
-      </Box>
+      <>
+        <title>{`${t('devPortal.title')} | ${t('profile.personalData.title')}`}</title>
```

Comment:
There's a linting error
```suggestion
        <title>{`${t('devPortal.title')} | ${t(
          'profile.personalData.title'
        )}`}</title>
```